src/components/TabContent.js

```diff
@@ -8,6 +8,10 @@ export const styles = {
 };
 
 class TabContent extends Component {
+    canRenderChildren () {
```

Comment:
Nit: can you please remove the space before parens to be in line with current style